downstream/modules/platform/proc-installing-the-ansible-platform-operator.adoc

```diff
@@ -0,0 +1,18 @@
+[id="proc-installing-the-ansible-platform-operator_{context}"]
+
+= Installing the {PlatformNameShort} Operator
+
+Use the following procedure to install the {PlatformNameShort} Operator.
+
+.Procedure
+. Navigate to menu:Operator[Operator Hub] and search for the {PlatformNameShort} Operator.
+. Select the {PlatformNameShort} Operator project.
+. Click on the Operator tile.
+. Click btn:[Install].
+. Select a Project to install the Operator into.
+Ansible recommends using the Operator recommended Namespace name.
+.. If  you want to install the Operator into a project other than the recommended one, select *Create Project* from the drop down menu.
+.. Enter the Project name.
+.. Click btn:[Create].
+.. Click btn:[Install].
```

Comment:
Should these be steps 6 and 7  rather than steps 5c and 5d?

Reply:
No. they're part of the installation process if you want to use a different project to the one recommended.

Reply:
The first one anyway.